src/color-picker/color-picker.scss

```
@@ -122,11 +122,22 @@ $iui-focus-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-primary-
   &-fields {
     display: flex;
     width: 100%;
+    gap: $iui-xxs;
 
-    .iui-input:not(:last-child) {
+    > *:not(:last-child) {
       margin-right: $iui-xxs;
     }
 
+    @supports (gap: $iui-xxs) {
+      > *:not(:last-child) {
+        margin-right: 0;
+      }
+    }
+
+    > * {
+      flex-grow: 1;
+    }
```

Can make it more readable
```suggestion
    > * {
      flex-grow: 1;

      &:not(:last-child) {
        margin-right: $iui-xxs;
        @supports (gap: $iui-xxs) {
          margin-right: 0;
        }
      }
    }
```